cmd/skopeo/layers.go

```diff
@@ -15,10 +17,14 @@ import (
 var layersCmd = cli.Command{
 	Name:      "layers",
 	Usage:     "Get layers of IMAGE-NAME",
-	ArgsUsage: "IMAGE-NAME",
+	ArgsUsage: "IMAGE-NAME [LAYER...]",
 	Action: func(c *cli.Context) error {
+		fmt.Println(`DEPRECATED: skopeo layers is deprecated in favor of skopeo copy `)
+		if c.NArg() == 0 {
+			return errors.New("please specify an image")
+		}
```

Comment:
ACK on having the check; for consistency, this should say `usage: correct usage`, like e.g. https://github.com/projectatomic/skopeo/blob/master/cmd/skopeo/delete.go#L12